source/faq.rst

```diff
@@ -0,0 +1,243 @@
+
+.. _sys-req:
+
+What are the minimum system requirements for PMM?
+================================================================================
+
+.. rubric:: |pmm-server|
+
+Any system which can run Docker version 1.12.6 or later.
+
+It needs roughly 1 GB of storage for each monitored database node
+with data retention set to one week.
+
+.. note::
+
+   By default, :ref:`retention <data-retention>` is set to 30 days for
+   Metrics Monitor and for Query Analytics.  Also consider
+   :ref:`disabling table statistics <performance-issues>`, which can
+   greatly decrease Prometheus database size.
+
+Minimum memory is 2 GB for one monitored database node, but it is not
+linear when you add more nodes.  For example, data from 20 nodes
+should be easily handled with 16 GB.
+
+.. rubric:: |pmm-client|
+
+Any modern 64-bit Linux distribution. It is tested on the latest
+versions of Debian, Ubuntu, CentOS, and Red Hat Enterprise Linux.
+
+Minimum 100 MB of storage is required for installing the |pmm-client|
+package.  With good constant connection to |pmm-server|, additional
+storage is not required.  However, the client needs to store any
+collected data that it is not able to send over immediately, so
+additional storage may be required if connection is unstable or
+throughput is too low.
+
+.. _data-retention:
+
+How to control data retention for PMM?
+================================================================================
+
+By default, both |prometheus| and QAN store time-series data for 30 days.
+
+Depending on available disk space and your requirements,
+you may need to adjust data retention time.
+
+You can control data retention by passing the :option:`DATA_RETENTION`
+environment variable when :ref:`creating and running the PMM Server container
+<server-container>`.  To set environment variable, use the ``-e``
+option.  The value should be the number of hours, and requires h suffix. 
+For example, the default value of 30 days for |opt.metrics-retention| is
+``720h``, but you can decrease the retention period for |prometheus| to 8 days
+as follows::
+
+-e DATA_RETENTION=192h
+
+.. seealso::
+
+   Metrics and queries retention
+      :option:`DATA_RETENTION`
+
+How often are nginx logs in PMM Server rotated?
+================================================================================
+
+|pmm-server| runs ``logrotate`` to rotate nginx logs on a daily basis
+and keep up to 10 latest log files.
+
+.. only:: showhidden
+
+	.. _performance-issues:
+
+	What are common performance considerations?
+	================================================================================
+
+	If a MySQL server has a lot of schemas or tables,
+	it is recommended to disable per table metrics when adding the instance:
+
+	.. prompt:: bash
+
+	   sudo pmm-admin add mysql --disable-tablestats
+
+	.. note:: Table statistics are disabled automatically
+	   if there are over 1 000 tables.
```

Comment:
not implemented for now

Reply:
This section is hidden (tab indentation after the .. only:: showhidden directive), but not removed to simplify its further inclusion